src/screens/WelcomeScreen.js

```diff
@@ -39,25 +36,36 @@ const styles = StyleSheet.create({
     justifyContent: "flex-end",
     alignItems: "center",
   },
-  loginButton: {
-    width: "100%",
-    height: 30,
-    backgroundColor: colors.primary,
+  textTitle: {
+    fontSize: 30,
+    fontWeight: "bold",
+  },
+  textDescription: {
+    fontSize: 18,
+    textAlign: "center",
+  },
+  textButton: {
+    fontSize: 20,
+    textAlign: "center",
+  },
+  button: {
```

Comment:
add 
justifyContent:"center"
to the style named button. It will make the text inside button locates in the middle.

Reply:
Just update with the changes